### ISSUE TITLE
Mark previous false positives

### ISSUE THREAD

**Issue by darthjuanan** [Te-k/phpscanner#5]
Is any way to tell the software that some files are always valid?

For example, I'm using a file in a plugin for WordPress, the JS-Composer, and the valid file:

wp-content/plugins/js_composer/vc_classmap.json.php

is always cataloged as malware.

Can I tell that this kind of files are valids?

Thanks in advance.

**Comment by Te-k** [comment]
No, it is not implemented, and I am not maintaining this tool anymore, but feel free to propose pull requests.
